content/about/history.md

```diff
@@ -4,91 +4,112 @@ date: 2018-12-29T11:02:05+06:00
+Several universities adopted GRASS as an important training and research 
+environment and many conducted short-courses for the public, in addition 
+to using GRASS in their own curricula. Indeed, in 1988, GRASS received an
+award for quality software from the Urban and Regional Information 
+Systems Association (URISA) for quality software. Only 4 years after 
+the first release.
 
-### Since 2000...
-From Dec 2000 to Dec 2006 the <a href="https://web.archive.org/web/20070630161517/http://intevation.de/rt/webrt?q_status=open&amp;q_queue=grass">GRASS Request Tracker (RT)</a> was used for <a href="https://grasswiki.osgeo.org/wiki/Bug_tracking">bug tracking</a>.
-
-In 2001, the GRASS Web site was moved to ITC-irst (Trento, Italy).
-
-In February 2006, the <a href="http://www.osgeo.org/">Open Source Geospatial Foundation</a> (OSGeo) was formed to support and promote worldwide use and collaborative development of Open Source geospatial technologies and data. It includes GRASS as one of its founding projects.
-
-From Jan 2007 to Dec 2008 GForge (a feature-rich system for collaborating on a software project) was used, operated by <a href="https://www.intevation.de">Intevation</a> company.
-
-On Dec 9, 2007 we started to use <a href="https://grasswiki.osgeo.org/wiki/GRASS_Migration_to_OSGeo">SVN</a> and <a href="https://trac.osgeo.org/grass">Trac</a> bugtracker hosted by OSGeo along with other GRASS GIS project infrastructure (Web-site, mailing lists) runs at OSGeo (see here for the <a href="/about/history/web-evolution">GRASS GIS Website evolution: 1997-2012</a>).
-In September 2006, the <a href="http://grasswiki.osgeo.org/wiki/PSC">GRASS Project Steering Commitee</a> was formed which is responsible for the overall management of the project. The PSC is especially responsible for <a href="http://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheMainGRASS-SVNrepository">granting SVN write access</a>.
-
-Birthday 2013: US Army Corps of Engineers - <a href="http://www.erdc.usace.army.mil/Media/News-Stories/Article/476565/grass-gis-turns-30-erdcs-cerl-was-there-at-the-start/">GRASS GIS turns 30 – ERDC’s CERL was there at the start.</a>
-
-Migration from OSGeo-SVN to <a href="https://github.com/OSGeo/grass">GitHub</a> was done in April/May 2019 (<a href="https://trac.osgeo.org/grass/wiki/GitMigration">details</a>).
+#### The 80's told by the main actors
 
 <ul id="links" class="version">
-  <li>
-  <span class="mwl">Scaling up globally: 30 years of FOSS4G development by Markus Neteler (Keynote at FOSS4G CEE 2013, 16-20 June, Bucharest, Romania)</span>
-  <a href="https://www.slideshare.net/markusN/scaling-up-globally-30-years-of-foss4g-development-keynote-at-foss4gcee-2013-romania" class="inl btn btn-primary" target="_blank">View it</a>
+ <li>
+ <a href="http://tldp.org/HOWTO/GIS-GRASS/history.html" target="_blank">A Brief History of GRASS</a> in the GRASS Mini-HOWTO by David A. Hastings, 1997.
```

Comment:
Silly question: isn't MD preferred here?

(if so, I can assist to convert)

Reply:
I have now changed most links to MD style. I know it looks cleaner, but html tags allow to open in a different tab, I don't know if we can do the same with MD.

If there are no further objections, I'll merge this and then we can improve styling